stressberry/cli.py

```diff
@@ -65,7 +71,22 @@ def _get_parser_run():
         "--cores",
         type=int,
         default=None,
-        help="number of cpu cores to stress (default: all)",
+        help="number of CPU cores to stress (default: all)",
+    )
+    parser.add_argument(
+        "-f", "--frequency", help="measure CPU core frequency", action="store_true"
```

Comment:
Remove this option -- always measure freq. (See comment below.)

Reply:
Initially, I didn't want to do this, as I saw in early versions of stressberry you had this capability but took it out. But as you're now happy I'll make it always record it.